README.md

```diff
@@ -33,16 +33,42 @@ python -m unittest discover
 
 Execute the following command from the project root directory to run program. The test_payload_for_parsed_json is optional parameter.
 
+### Install DITA-OT
+
+Before you can use the dita command, you need to download and install the `DITA-OT` package on your system. You can download the latest version of `DITA-OT` from the official [website](https://www.dita-ot.org/download). Once you have downloaded and extracted the package to a directory on your system, you can add the `DITA-OT/bin` directory to your `PATH` variable by adding the following line to your `.bash_profile file`:
```

Comment:
second sentence of this paragraph is wrong - it should be explaining that we're about to check if dita is installed.

Reply:
@IanMayo I have added an order for the instructions, i hope this makes it clear, please let me know if there is anything i have to fix